cores/esp8266/core_esp8266_i2s.cpp

```diff
@@ -493,6 +494,10 @@ float i2s_get_real_rate(){
 }
 
 bool i2s_rxtx_begin(bool enableRx, bool enableTx) {
+  return i2s_rxtxdac_begin(enableRx, enableTx, true);
+}
+
+bool i2s_rxtxdac_begin(bool enableRx, bool enableTx, bool txDAC) {
```

Comment:
I think the naming/API needs to be a bit more generic.  Maybe make it `i2s_rxtxdrive_begin(bool enRx, bool enTx, bool drRxClocks, bool drTxClocks`.  There are other reasons to use I2S w/o the other pins than DAC/NoDAC (IR blaster, video out, RGB control, etc.).  The idea here is we're potentially disabling the clocking drives.